www/src/css/custom.css

```diff
@@ -279,6 +279,12 @@ h5 {
   display: none;
 }
 
+/* Would be nice to have these displayed but they are veeerry long and noisy... */
+.shiki .error,
+.shiki .error-behind {
+  display: none;
+}
+
```

Comment:
We shouldn't have errors tho should we

It's possible to surpress the errors with `@noErrors` or some directive like that

Reply:
Yes but in some cases it's nice to show the red squiggly - which you can do by expecting a certain error. It was just the text which was soooo long that I hide it.

Reply:
i believe so, let me try

Reply:
![image](https://user-images.githubusercontent.com/51714798/212165290-a72f730f-dcb1-4703-9555-62c95334d020.png)

Reply:
I see that's really nice, but aren't we abusing something here that really should prevent the website from even building though? 🤔

Reply:
So I'm expecting that line to throw a certain error - if in the future it would throw a new error which isn't expected - the build would fail.

https://github.com/trpc/trpc/pull/3489/files#diff-51f1020e33badf07d07e0179b736fe32eac31630b6cc82b182483b683bc7b16bR218

Here's from the docs: 

![CleanShot 2023-01-12 at 20 57 17](https://user-images.githubusercontent.com/51714798/212168303-c2c26d8b-a482-48f8-b916-8b51555630f0.png)

Reply:
this should be okay 👍🏻